src/format.rs

```diff
@@ -313,7 +313,13 @@ impl Documentable for &UntypedPattern {
         match self {
             Pattern::Int { value, .. } => value.clone().to_doc(),
 
-            Pattern::Float { value, .. } => value.to_doc(),
+            Pattern::Float { value, .. } => {  
+                if value.ends_with(".") {
+                    format!("{}0", value).to_doc()
+                } else {
+                    value.clone().to_doc()
+                }
```

Comment:
I'm not sure if thats a good approach. Also for some reason `value.ends_with(".")` always returns false

Reply:
Sorry, I have directed you to the wrong file. We want the Erlang printer, not the Gleam formatter.

Reply:
np, fixed 😸